### DISCUSSION TITLE
Thank you for your hard work!

### DISCUSSION THREAD

**Discussion by lpochron** [mantinedev/mantine#6918] [Kind words]
Hi,

Now after the brief period of initial setup, a couple days of awkwardness with new syntax and having made a few layouts...

Mantine is so much better than whatever I ever tried! Very complete, friction-less and well-thought. So many things that the contributors thought about. The forms are just getting the job done, notifications - easy to use. Docs contain so many examples that actually look great. You even thought about small details like persisting the state of dark theme. So clever. And everything looks nice just out of the box. Even on a most basic markup!

This library tells me a story about the wealth of UI experience that you have. Mantine knows exactly what developers would like to skip - so we can move on to some more pragmatic stuff. It's pleasure to use this absolute gem! 👍 

Thank you! x 100

**Comment by rtivital** [comment]
Thank you for kind words!
